Code/Source/svFSI/ComMod.h

```diff
@@ -1457,7 +1458,7 @@ class ComMod {
 
     /// @brief Total number of nodes (number of nodes on current proc across
     /// all meshes)
-    int tnNo = 0;
+    int tnNo = 0; // total number of nodes across all meshes, but only on current processor
```

Comment:
In-line comments clutter the  code, use better variable names or document these in the function comments.